docs/docs/connect/data-source/https.md

```diff
@@ -12,17 +12,29 @@ Import data from remote sources accessible via HTTP or HTTPS URLs into your Rill
 The HTTPS connector allows you to import data from publicly accessible files hosted on web servers, CDNs, or cloud storage services. This is perfect for working with datasets that are regularly updated or shared publicly.
 
 
+## Connect to HTTPS
+
+If your endpoint requires authentication, you can create a https connector that can test your API key before getting started on importing data. 
```

Comment:
It feels like this is missing the case for "If your endpoint _does not_ require authentication".